layers/meta-resin-beaglebone/recipes-support/resin-init/resin-init-flasher.bbappend

```diff
@@ -1,9 +1,2 @@
 # Flash to internal eMMC - kernel enumerates this, when SD card is in, as mmcblk1
 INTERNAL_DEVICE_KERNEL_beaglebone = "mmcblk1"
-
-# the name of the bootloader config file to be written to internal media
-INTERNAL_DEVICE_BOOTLOADER_CONFIG_beaglebone = "uEnv.txt_internal"
-
-# the pathname (relative to /boot/) where the internal bootloader config file should reside in
-# (currently assuming this is on partition 1 of the internal media)
-INTERNAL_DEVICE_BOOTLOADER_CONFIG_PATH_beaglebone = "/uEnv.txt"
```

Comment:
Shall we also remove now the code in the flasher script that handles these vars?